doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -208,7 +208,9 @@ security-scopes: https://yourendpoint.azure.com/.default
 - Support both credentials(e.g. `-securityHeaderName` and `--securityScope` are specified):
 
 ```yaml
-security: AADToken;AzureKey
+security:
+  - AADToken
```

Comment:
It is need that `-` align with `security`

Reply:
How to align? I tried this configuration. It works well.

Reply:
ok.